src/sentry/utils/dates.py

```diff
@@ -72,16 +72,16 @@ def parse_timestamp(value):
         return value
     elif isinstance(value, six.integer_types + (float,)):
         return datetime.utcfromtimestamp(value).replace(tzinfo=pytz.utc)
-    value = (value or "").rstrip("Z").encode("ascii", "replace").split(".", 1)
+    value = (value or "").rstrip("Z").encode("ascii", "replace").split(b".", 1)
```

Comment:
I'm not totally clear why we encode in the first place here... We decode almost immediately afterwards. Maybe it's just so that we replace any bad chars?

Reply:
I addded that decode after, but yeah.. I'm not sure why it was encoded either. 😬